src/languages/es.ts

```diff
@@ -3268,6 +3268,12 @@ export default {
                             return 'Actualizando información de conexión';
                         case 'netSuiteSyncNetSuiteReimbursedReports':
                             return 'Marcando informes de Expensify como reembolsados';
+                        case 'netSuiteSyncImportCustomLists':
+                            return 'Importando listas personalizado';
+                        case 'netSuiteSyncImportSubsidiaries':
+                            return 'Importando subsidiarias';
```

Comment:
Also this one, sounds like `subsidiario/a` is more commonly used as an adjective, whereas the noun would be `filial`.

Reply:
`subsidiarias` is what NetSuite uses in product when you turn it into Spanish 🤷🏽‍♂️